Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -225,7 +229,9 @@ open class CarPlayMapViewController: UIViewController {
     }
     
     func setupNavigationMapView() {
-        let navigationMapView = NavigationMapView(frame: UIScreen.main.bounds, navigationCameraType: .carPlay)
+        let navigationMapView = NavigationMapView(frame: UIScreen.main.bounds,
```

Comment:
Maybe we should consider changing the default value of `tileStoreLocation ` in `NavigationMapView` initializer, so it uses the value from `NavigationSettings.shared.tileStoreConfiguration.mapLocation`? 🤔

Reply:
I was thinking about that. But is it correct from NavigationMapView's logic perspective? Why would a default value of map view depend on configuration of some other singleton?
I decided that `.default` is a good default init value, while Mapbox provided components do setup with NavigationSettings.